test/perf/micro/bin/table.pl

```diff
@@ -12,72 +12,72 @@
 our @benchmarks = qw(
   iteration_pi_sum
   recursion_fibonacci
-  recursion_quicksort  
+  recursion_quicksort
   parse_integers
   print_to_file
   matrix_statistics
   matrix_multiply
   userfunc_mandelbrot
 );
 
-our $c_ver = `gcc -v 2>&1 | grep "gcc version" | cut -f3 -d" "`;
-our $julia_ver = `../../../julia -v | cut -f3 -d" "`;
-our $fortran_ver = `gfortran -v 2>&1 | grep "gcc version" | cut -f3 -d" "`;
-our $python_ver = `python3 -V 2>&1 | cut -f2 -d" "`;
-our $matlab_ver = `matlab -nodisplay -nojvm -nosplash -r "version -release, quit" | tail -n 3 | head -n 1`;
-our $R_ver = `R --version | grep "R version" | cut -f3 -d" "`;
-our $octave_ver = `octave -v | grep version | cut -f4 -d" "`;
-our $go_ver = `go version | cut -f3 -d" "`;
-#our $lua_ver = `scilua -v 2>&1 | grep Shell | cut -f3 -d" " | cut -f1 -d,`;
+chomp(our $c_ver = `gcc -v 2>&1 | grep "gcc version" | cut -f3 -d" "`);
+chomp(our $julia_ver = `../../../julia -v | cut -f3 -d" "`);
+chomp(our $fortran_ver = `gfortran -v 2>&1 | grep "gcc version" | cut -f3 -d" "`);
+chomp(our $python_ver = `python3 -V 2>&1 | cut -f2 -d" "`);
+chomp(our $matlab_ver = `matlab -nodisplay -nojvm -nosplash -r "version -release, quit" | tail -n 3 | head -n 1`);
+chomp(our $R_ver = `R --version | grep "R version" | cut -f3 -d" "`);
+chomp(our $octave_ver = `octave -v | grep version | cut -f4 -d" "`);
+chomp(our $go_ver = `go version | cut -f3 -d" "`);
+#chomp(our $lua_ver = `scilua -v 2>&1 | grep Shell | cut -f3 -d" " | cut -f1 -d,`);
 our $lua_ver = "scilua v1.0.0-b12"; # scilua has no run-time versioninfo function
-our $javascript_ver = `nodejs -e "console.log(process.versions.v8)"`;
-our $mathematica_ver = `echo quit | math -version | head -n 1 | cut -f2 -d" "`;
-#our $stata_ver = `stata -q -b version && grep version stata.log | cut -f2 -d" " && rm stata.log`;
-our $java_ver = `java -version 2>&1 |grep "version" | cut -f 3 -d " " | cut -c 2-9`;
+chomp(our $javascript_ver = `nodejs -e "console.log(process.versions.v8)"`);
+chomp(our $mathematica_ver = `echo quit | math -version | head -n 1 | cut -f2 -d" "`);
+#chomp(our $stata_ver = `stata -q -b version && grep version stata.log | cut -f2 -d" " && rm stata.log`);
+chomp(our $java_ver = `java -version 2>&1 |grep "version" | cut -f3 -d " " | cut -c 2-9`);
```

Comment:
I would put the `chomp` around the command rather than the declaration. This works because strings are mutable in Perl, but it's a bit weird to assign the string and then mutate it rather than just chomp it and then assign the chomped value.

Reply:
Sure -- I did find it odd too, but I don't know what is expected as idiomatic Perl, and that's the style I found in the example I based this on, so I went with that. I'll change in a bit.